specification/columns/contractappliedcost.md

```diff
@@ -0,0 +1,41 @@
+# Contract Applied Cost
+
+Contract Applied Cost represents the cost of the charge applied to the contract.  Contract Applied Cost is applied to the contract via [Contract ID](#contractid).  Contract Applied Cost is commonly used for monitoring the progress towards fulfilling contractual commitments that facilitate discounts for [*resources*](#glossary:resource) or [*services*](#glossary:service) as negotiated between a provider and a customer.
```

Comment:
nit: "Contract Applied Cost is applied to the contract..." presents itself as self-defining, but I'm not sure it is. I think I understand what this is covering... One possible idea:
```suggestion
Contract Applied Cost is the portion of a charge counted toward a specific contract, identified by its [Contract ID](#contractid). Contract Applied Cost is commonly used for monitoring the progress towards fulfilling contractual commitments that facilitate discounts for [*resources*](#glossary:resource) or [*services*](#glossary:service) as negotiated between a provider and a customer.
```

Fwiw, it's not really applied to the contract. It's applied to a commitment.